src/components/FeedbackContent.js

```diff
@@ -65,6 +80,15 @@ const styles = StyleSheet.create({
     shadowColor: COLORS.textColor,
     shadowOpacity: 0.08,
   },
+  offMargin: {
+    marginTop: 0,
+    marginBottom: 0,
+    marginRight: 0,
+    marginLeft: 0,
```

Comment:
```suggestion
    margin: 0,
```

Reply:
It is not given that `margin: 0` will work here, see this issue I have opened about _flatten_ styles in React Native: https://github.com/HathorNetwork/hathor-wallet-mobile/issues/532